ush/setup.sh

```diff
@@ -1239,6 +1239,7 @@ FIXam="${EXPTDIR}/fix_am"
 FIXLAM="${EXPTDIR}/fix_lam"
 FIXgsi="${EXPTDIR}/fix_gsi"
 FIXcrtm="${EXPTDIR}/fix_crtm"
+SST_ROOT="/lfs4/BMC/public/data/grids/ncep/sst/0p083deg/grib2"
```

Comment:
SST_ROOT is hardcoded in setup.sh and it only considers the Jet situation. Do we want to put it in config.sh so that it is configurable for different situations?

Reply:
Good point! Will add it to config.sh. Thanks!